src/databricks/labs/remorph/helpers/file_utils.py

```diff
@@ -102,3 +102,20 @@ def refactor_hexadecimal_chars(input_string: str) -> str:
     for key, value in highlight.items():
         output_string = output_string.replace(key, value)
     return output_string
+
+
+def format_error_message(error_type: str, error_message: Exception, error_sql: str) -> str:
```

Comment:
this is technically string utils can you move it to a new file `string_utils.py`